src/components/pages/ImageClassification/ImageAnnotationModal/ImageAnnotationModal.styles.js

```diff
@@ -114,16 +114,16 @@ export const PopupTdForRadio = styled(PopupTd)`
   width: 15px;
```

Comment:
To summarize this review. I renamed a couple of things and most jsut moved existing code to different files. I will highlight large chunks of copy paste